BuildProject/App.tsx

```diff
@@ -0,0 +1,139 @@
+/**
+ * Promoted React Native Metrics testing app.
+ *
+ * @format
+ * @flow strict-local
+ */
+
+import PromotedMetrics, { ActionType } from '@promotedai/react-native-metrics';
+import React, { useState } from 'react';
+import type { Node } from 'react';
+import {
+  Alert,
+  Button,
+  SafeAreaView,
+  ScrollView,
+  StatusBar,
+  Text,
+  useColorScheme,
+} from 'react-native';
+
+import {
+  Colors,
+} from 'react-native/Libraries/NewAppScreen';
+
+const App: () => Node = () => {
+  const isDarkMode = useColorScheme() === 'dark';
+
+  const backgroundStyle = {
+    backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
+  };
+
+  const [text, setText] = useState('');
+
+  return (
+    <SafeAreaView style={backgroundStyle}>
+      <StatusBar barStyle={isDarkMode ? 'light-content' : 'dark-content'} />
+      <ScrollView
+        contentInsetAdjustmentBehavior="automatic"
+        style={backgroundStyle}>
+        <Button 
+          title="Test All" 
+          onPress={() => {
```

Comment:
nit - Might be easier to read if you move the onPress to a separate method instead of embedding it.

Reply:
Done.